server.js

```diff
@@ -1,27 +1,23 @@
-const express = require('express')
-const cors = require('cors');
-const inventoriesRoutes = require('./routes/inventoriesRoutes');
-const app = express()
-const port = 3000
+const express = require("express");
+const cors = require("cors");
+const inventoriesRoutes = require("./routes/inventoriesRoutes");
+const warehouseRoutes = require("./routes/warehouseRoutes");
+const app = express();
+const port = 3000;
 
 // Middleware
-app.use(express.json()) 
-app.use(cors()); 
-app.use(express.static('public'))
+app.use(express.json());
+app.use(cors());
+app.use(express.static("public"));
 
-
-
-
-
-app.get('/', (req, res) => {
-  res.send('Hello World!')
+app.get("/", (req, res) => {
+  res.send("Hello World!");
 });
 
-
-// Routes 
-app.use('/inventories/:warehouse_id', inventoriesRoutes);
-
+// Routes
+app.use("/inventories/", inventoriesRoutes);
```

Comment:
why did you change this line of code from app.use('/inventories/:warehouse_id', inventoriesRoutes);  to  app.use("/inventories/", inventoriesRoutes);

Reply:
Unintentionally. Probably changed it whilst dealing with warehouses routes.

Reply:
oh ok then you need to change it back cox to add an inventory you need to be able to know what warehouse that inventory belongs to that why i added the warehouse_id to the route